builtin/providers/aws/resource_aws_route53_record_test.go

```diff
@@ -104,6 +104,9 @@ func testAccCheckRoute53RecordExists(n string) resource.TestCheckFunc {
 }
 
 const testAccRoute53RecordConfig = `
+provider "aws" {
+    region = "us-east-1"
+}
```

Comment:
For posterity: fixed in 700ce30f by hardcoding `us-east-1`, because v4 of Signing requires all global things (like Route 53) to use `us-east-1`.